cpp/test/Slice/errorDetection/IdentAsKeyword.ice

```diff
@@ -41,7 +41,6 @@ module Test
 
     dictionary<long, long> throws;
     dictionary<long, long> thRows;
-    dictionary<long, long> LOCALobject; // Ok as of 3.7
```

Comment:
This was testing how we handled the `LocalObject` type.
`LocalObject` is no longer a keyword, so no point to this.